### ISSUE TITLE
[Northrend] [The Drakkensryd - quest doesn't work]

### ISSUE THREAD

**Issue by Nextwarlock** [dalaranwow/dalaran-wow#4437]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. The quest "The Drakkensryd" from the realm Kirin Tor is full of bugs! It can not be played.
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
reworked the whole questscript and paths here https://github.com/dalaranwow/dalaran-wow/issues/4349

the fix needs to be tested on PTR.

**Comment by Nextwarlock** [comment]
Sorry, but it's nothing changed!
Is it possible to see this quest as here? 
[https://www.youtube.com/watch?v=oCcyIi7sqZE](url)
I don't get any proto-drake when I start the quest, I see riders falling from sky when I arrive there, I jump on the back of one prot-drake and I try to fight with the rider, but I get the message Out of space, I target one rider from my front and with luck I kill him, but it doesn't count.... What is it working in this quest? other than the harpoon.

**Comment by Rushor** [comment]
no the quest is fixed on the testserver of the developer the fix for it will be applied on the live server in the future. 

read https://github.com/dalaranwow/dalaran-wow/blob/master/README.md before posting issues. thanks

**Comment by Nextwarlock** [comment]
Hello,
Thanks for your quick reply. 
I don't know , but do I have to play on the testserver? 
No updates applied lately in Kirin Tor, more players of this realm complain the Agalon has very good pve quests, updated and without bugs,but here we are down with many bugged quests.
When I saw the problem is fixed first time then i thought it's ok on server, too. I'm a player, not a programmer.
What does "it will be applied on the live server in the future." mean? Today? This week? This month? This year? ...
Best regards

**Comment by Rushor** [comment]
> What does "it will be applied on the live server in the future." mean? Today? This week? This month? This year? ...

I'm not respnsible for the apply of the fixes on live, i ony create the fixes. according to http://www.dalaran-wow.com/news/101/a-quick-note-about-the-icecrown-citadel-and-changelogs/ it will be in the next months as far as i know.
